infra/growth/campaigns/august.js

```diff
@@ -264,6 +268,8 @@ const augustConfig = {
             eventType: 'SharedOnTwitter',
             additionalLockConditions: ['TwitterAttestation'],
             content: {
+              // different social network shares sharing same content should have the same id
```

Comment:
It was not immediately obvious to me what the id was used for.
I think I understand it is used for a) grouping in the DApp and b) to log event in case of a non-verifiable sharing. Couple suggestions:
 - explain this in the comment
 - add a unit test for august rules that checks that for rules with same id, the content is the same. I'm sure you defined it correctly in august but may help us catch discrepancies in rules config in future campaigns :)

Reply:
Another thought would be to take the content definition out of the rules and move it up as a separate section in the config. So that it is defined once and does not need to be duplicated in every rule. A rule would only need to have the content.id in its config.
It would look like:
```
const augustConfig = {
  numLevels: 3,
  levels: {
    0: {
      rules: [
      ....
   }
   content: [
   {
     id: "origin_app",
     titleKey: ...
   },
   ....
   ]
```

Though the downside is that in the future the content may vary depending on the social network ? Like tweets are short and include hashtags while facebook posts can be longer and have no hashtags. So maybe that's not a good idea...

Reply:
That's a good point. More social networks is going to bring only more redundancy in configuration. Though about it a bit... really like your idea with separate content config. So have changed it to that and separated the post contents from:
```
content: {
  id: "origin_app"
  ....
  post: {
    text: {...}
  }
}
```

to

```
content: {
  id: "origin_app"
  ....
  post: {
    tweet: {...}
  }
}
```
Since the current post config is really applicable only to Twitter. And in case we want to add other more generic sharing contents besides `tweet` like `short_text` or `text` that can be reused in multiple social networks we can.

Do you like this approach?

Reply:
Ah yes, good thinking. I like your idea of having different fields if necessary in content, like tweet, post, etc.. Sounds good to me ! :)